index.rst

```diff
@@ -1,51 +1,212 @@
+2. ``ingestCalibs.py``, from the ``pipe_tasks`` package,
+3. ``processCcd.py``, from the ``pipe_tasks`` package, and
+4. ``imageDifference.py``, also from the ``pipe_tasks`` package.
+
+To streamline this process, we wrote a python script called ``decam_process.py`` to handle each step.
+It is `available on GitHub <https://github.com/lsst-dm/decam_hits/blob/master/decam_process.py>`_.
+While it was only used on the :ref:`Dataset <Dataset>` described above, it can
+in principle be applied to any set of raw DECam images using the LSST Stack, as shown below.
+
+
+.. _Tutorial:
+
+Tutorial
+--------
+
+The prerequisites for running the Prototype Pipeline on a dataset of images (\*.fits or \*.fits.fz) are:
+
+- The LSST Stack with the standard modules setup, as well as ``obs_decam``
+- A directory containing some raw DECam images
+- A directory containing DECam MasterCal biases and flats corresponding to the raw images
+- A directory containing DECam defect images, one for each CCD
+- A set of astrometric reference catalogs (example below)
+- A personal working copy of the `decam_process <https://github.com/lsst-dm/decam_hits/blob/master/decam_process.py>`_ script
+
+To begin, edit the values near the top of the ``decam_process`` script to reflect your desired directory names
+for where processed images will live, as well as to specify the visits and CCDs you wish to process. The default presets are
+
+.. code-block:: python
+
+   repo = 'ingested_15A38/'  # used by ingest, ingestCalibs, processCcd
+   calibrepo = 'calibingested_15A38/'  # used by ingestCalibs, processCcd
+   processedrepo = 'processed_15A38/'  # used by processCcd, diffIm
+   diffimrepo = 'diffim_15A38_g/'  # used by diffIm
+   visits = [410927, 411033]  # used by processCcd, diffIm
+   # NOTE: visits assumes the first element is template and the rest are science
+   ccdnum = '1..62'  # used by processCcd, diffIm
+   # NOTE: the default '1..62' value includes all of the DECam CCDs
+
+In general, ``repo`` refers to where ingested images will live, ``calibrepo``
+refers to where ingested calibration products (flats and biases) will live, ``processedrepo`` refers to where "calexp"
+images will live (i.e., those that have been processed with ``processCcd`` including steps 1 through 3 in :ref:`Overview <Overview>`),
+and ``diffimrepo`` refers to where difference images and DIA Sources (catalogs) will ultimately live.
+Visit numbers can be found in image headers or retrieved from the registry database created in ``repo`` during image ingestion 
+(visit numbers are not used during ingestion, so you may set them after this step). The first visit you specify
+in the list will be used as the template.
+
+Finally, copy or link the Pan-STARRS astrometric reference catalog into the directory you've chosen for 
+``repo`` and call it ``ref_cats``. If you are working on the ``lsst-dev`` server, you can link the Gaia, 
+Pan-STARRS, and SDSS catalogs by 
+
+.. prompt:: bash
+   
+   mkdir repo
```

Comment:
Maybe better to say "using the default dir. for `repo` (`ingested_15A38/`) one would do:

and then `mkdir ingested_15A38/`.  (Instead of `mkdir repo`).